cmd/config.go

```diff
@@ -40,12 +40,12 @@ const (
 	minioConfigBackupFile = minioConfigFile + ".backup"
 )
 
-func saveServerConfigIndent(ctx context.Context, objAPI ObjectLayer, config *serverConfig) error {
+func saveServerConfigStruct(ctx context.Context, objAPI ObjectLayer, config *serverConfig) error {
```

Comment:
saveServerConfigStruct is a bad name with `struct` in it, but could not do better